.github/workflows/docker-edge.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           context: .
           push: ${{ github.event_name != 'pull_request' }}
           file: docker/edge-${{ matrix.os }}.Dockerfile
-          platforms: linux/amd64,linux/arm64,linux/arm/v7
+          platforms: linux/amd64,linux/arm64,linux/arm/v7,linux/arm/v6
```

Comment:
This should be gated on `matrix.os` being `alpine` — see the build failure for Ubuntu.

Reply:
I'm new to this GitHub action thing (actually GitHub in general), I don't know how to do that, should I just change edge workflow to match release? 

On release, it have 2 action separated, not using matrix.os

Reply:
```suggestion
          platforms: linux/amd64,linux/arm64,linux/arm/v7${{ matrix.os == 'alpine' && ',linux/arm/v6' || '' }}
```